src/components/tasks.tsx

```diff
@@ -0,0 +1,95 @@
+import { Trash } from 'lucide-react'
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog'
+import { Button } from '@/components/ui/button'
+import { Checkbox } from '@/components/ui/checkbox'
+import { Label } from '@/components/ui/label'
+// import { ToastAction } from '@/components/ui/toast'
+import { useToast } from '@/components/ui/use-toast'
+
+interface TasksProps {
+  content: { content: string; completed: boolean }
+  onDeleteTask: (task: string, completed: boolean) => void
+  onToggleComplete: (task: string, completed: boolean) => void
+}
+
+export function Tasks({ content, onDeleteTask, onToggleComplete }: TasksProps) {
+  const { toast } = useToast()
+
+  const { content: taskContent, completed } = content
+
+  // function handleDeleteTask() {
+  //   toast({
+  //     title: 'Tarefa',
+  //     description: 'A tarefa foi excluída',
+  //     // action: <ToastAction altText="Desfazer">Desfazer</ToastAction>,
+  //   })
+
+  //   onDeleteTask(taskContent, completed)
+  // }
+
+  // function handleCheckboxChange() {
+  //   onToggleComplete(taskContent, !completed)
+  // }
```

Comment:
Se for usar os comentários ainda, é uma boa colocar um TODO comentado no código. Se não acho que da pra remover, mesma coisa pros outros comentários